units/provisioning_user_data_done_test.go

```diff
@@ -83,10 +83,10 @@ func TestUserDataDoneJob(t *testing.T) {
 				Host: "localhost",
 				Distro: distro.Distro{
 					BootstrapSettings: distro.BootstrapSettings{
-						Method:        distro.BootstrapMethodUserData,
-						Communication: distro.CommunicationMethodRPC,
-						ClientDir:     "/client_dir",
-						ShellPath:     "/shell_path",
+						Method:          distro.BootstrapMethodUserData,
+						Communication:   distro.CommunicationMethodRPC,
+						JasperBinaryDir: "/jasper_binary_dir",
```

Comment:
Super nit, but will this test fail locally once people upgrade to Catalina?

Reply:
It should work regardless of platform - this test uses a mock process manager so invoking the job doesn't create any processes.